corehq/apps/domain/models.py

```diff
@@ -51,8 +51,6 @@
 BUSINESS_UNITS = [
     "DSA",
     "DSI",
-    "DLAC",
-    "DMOZ",
```

Comment:
it looks like this is used as a `choices` argument on `InternalProperties`. are we sure that either 1) there are no domains in the DB with this set, or 2) that bad things won't happen if there are? looking quickly at `jsonobject` it seemed like this would raise an error everytime `validate` (and therefore `to_json`) is called.